packages/mermaid/src/diagrams/xychart/parser/xychart.jison

```diff
@@ -0,0 +1,168 @@
+%lex
+%options case-insensitive
+
+%x string
+%x md_string
+%x title
+%x acc_title
+%x acc_descr
+%x acc_descr_multiline
+%s axis_data
+%s axis_band_data
+%s data
+%s data_inner
+%%
+\%\%(?!\{)[^\n]*                          /* skip comments */
+[^\}]\%\%[^\n]*                           /* skip comments */
+<axis_data>(\r?\n)                        { this.popState(); return 'NEWLINE'; }
+<data>(\r?\n)                             { this.popState(); return 'NEWLINE'; }
+[\n\r]+                                   return 'NEWLINE';
+\%\%[^\n]*                                /* do nothing */
+
+"title"                                   { return 'title'; }
+
+"accTitle"\s*":"\s*                         { this.pushState("acc_title");return 'acc_title'; }
+<acc_title>(?!\n|;|#)*[^\n]*              { this.popState(); return "acc_title_value"; }
+"accDescr"\s*":"\s*                         { this.pushState("acc_descr");return 'acc_descr'; }
+<acc_descr>(?!\n|;|#)*[^\n]*              { this.popState(); return "acc_descr_value"; }
+"accDescr"\s*"{"\s*                         { this.pushState("acc_descr_multiline");}
+<acc_descr_multiline>"{"                 { this.popState(); }
+<acc_descr_multiline>[^\}]*               { return "acc_descr_multiline_value"; }
```

Comment:
I cannot find any place in grammar where this would we used. Is it necessary? It seems to be copy and paste from other grammar file.

Reply:
I might be mistaken but I think these involve accessibility features (e.g., text alternatives for diagrams maybe). We should definitely look into implementing these if Mermaid is serious about inclusivity

Reply:
Added the implementation